cmake/vcpkghelpers.cmake

```diff
@@ -120,7 +121,15 @@ function(ivw_vcpkg_install name)
     endif()
 
     if(NOT Python3_Interpreter_FOUND)
-        return()
+        find_package(Python3 COMPONENTS Interpreter)
+        if(NOT Python3_FOUND OR 
+           NOT Python3_Interpreter_FOUND)
+            message(ERROR "Python3 not available.\n"
```

Comment:
I don't want to make this a hard error.  The information gathered here is only needed for the about dialog/license system and the installation. Hence this should not be a hard error. Most people building inviwo won't need this...

Reply:
Good point.